distutils/unixccompiler.py

```diff
@@ -231,7 +231,7 @@ def runtime_library_dir_option(self, dir):
         # this time, there's no way to determine this information from
         # the configuration data stored in the Python installation, so
         # we use this hack.
-        compiler = os.path.basename(sysconfig.get_config_var("CC"))
+        compiler = os.path.basename(sysconfig.get_config_var("CC").split(" ")[0])
```

Comment:
Have you considered using [shlex.split](https://docs.python.org/3/library/shlex.html#shlex.split)? Although there's no specified standard for `CC`, relying on `shlex.split` would at least honor a very common convention and support both `"this path to compiler" --arg` and `this\ path\ to\ compiler --arg`.

```suggestion
        compiler = os.path.basename(shlex.split(sysconfig.get_config_var("CC"))[0])
```

Reply:
That's a great suggestion.